### PR TITLE
fix/ added .md to internal routes and fixed broken links

### PR DESCRIPTION
**Pull request by dennisvankekem**

### CODE REVIEW

File: docs/for-devs/get-started/lab-1.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ We assume you (or the platform administrator) have:
 - Trivy Operator
 - Gatekeeper (with option `Disable validating webhook` set to true)
 
-For the [Use OpenTelemery](lab-27) Lab, the Tempo app needs to be enabled together with tracing in the `Istio` and `Nginx Ingress` apps.
+For the [Use OpenTelemery](lab-27.md) Lab, the Tempo app needs to be enabled together with tracing in the `Istio` and `Nginx Ingress` apps.
 
 3. Created a team called `demo` with `Managed prometheus and alert manager` enabled
 4. [Created an account](/docs/apps/keycloak#create-a-user-in-keycloak) and added your user to the team group in Keycloak
```
Comment (ElderMatt) [inline docs/for-devs/get-started/lab-1.md:28]: what about this one?

File: docs/get-started/activation.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ If you would like to use the Otomi Console (Web UI) and Otomi API (for code vali
 - Go to [Otomi Cloud](https://portal.otomi.cloud)
 - Create an account
 - Click on `Register Cluster`
-- Agree with the [EULA](../../product/eula/v1)
+- Agree with the [EULA](/product/eula/v1)
```
Comment (ElderMatt) [inline docs/get-started/activation.md:30]: does this also need .md?

Comment (dennisvankekem) [reply docs/get-started/activation.md:30]: yes

File: docs/for-devs/get-started/lab-1.md (hunk 1 above)
Comment (dennisvankekem) [reply docs/for-devs/get-started/lab-1.md:28]: the `#create-a-user-in-keycloak` ? For some reason things with a refference in the page with # don't need it

File: docs/get-started/activation.md (hunk 2 above)
Comment (dennisvankekem) [reply docs/get-started/activation.md:30]: for some reason links from and to /product reject `.md`, but it seems to work even with `/` appended. I'll let it slide for the time being as it already costed too much time

**Review by ElderMatt — APPROVED** [review]
LGTM

**Pull request merged**